execution_workflows/Roar/README.md

```diff
@@ -0,0 +1,24 @@
+# APA-Scan
```

Comment:
Most comments on APA-Scan apply to Roar as well. Perhaps it would indeed be good to prepare another branch for Roar as this PR is quite unwieldy and some copy/paste artefacts crept in, so it's a bit risky to merge these two features into one PR. Please ask on Slack if you need help to disentagle that

Reply:
I agree, I'm afraid the two methods should really be on separate PRs